lib/Parse/ParseExpr.cpp

```diff
@@ -1744,8 +1744,7 @@ DeclName Parser::parseUnqualifiedDeclName(bool afterDot,
   // Consume the base name.
   Identifier baseName = Context.getIdentifier(Tok.getText());
   SourceLoc baseNameLoc;
-  if (Tok.is(tok::identifier) || Tok.is(tok::kw_Self) ||
-      Tok.is(tok::kw_self)) {
+  if (Tok.isAny(tok::identifier, tok::kw_Self, tok::kw_Self)) {
```

Comment:
Thanks, I’ve fixed this — I only managed to get the tests running today so a commit with passing tests is incoming.